src/app/examples/examples.common.ts

```diff
@@ -64,6 +64,4 @@ export const COMPONENT_DECLARATIONS: any[] = [
   AlertExampleComponent,
 ];
 
-export const PROVIDER_DECLARATIONS: any[] = [
-  { provide: CUSTOM_FONT_SETTINGS, useValue: customIconSettings },
-];
+export const PROVIDER_DECLARATIONS: any[] = [{ provide: ICON_SETTINGS, useValue: iconSettings }];
```

Comment:
Either remove (Kirby icons should come out of the box) or replace with example on custom icon settings/font